src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -292,6 +292,7 @@
                     profilerCell.appendChild(profilerLink);
                 }
 
+                broadCastToken(request.profile, "ajax", request.url);
```

Comment:
```suggestion
                if (request.profilerUrl) {
                    broadCastToken(request.profile, "ajax", request.url);
                }
```

Reply:
Good catch ! I didn't think about element that was not profiled.

I've made the change and moved the block to be near the logic of reloading the toolbar if needed, so that we only trigger once the broadcasting. I check for request.profile instead of request.profilerUrl to be more consistent with the parameter of the function and the toolbar replace behavior